VimeoNetworking/Sources/ResponseCache.swift

```diff
@@ -273,13 +278,17 @@ final internal class ResponseCache
         
         private func cachesDirectoryURL() -> NSURL
         {
+            // Apple /Caches directory
             guard let directory = NSSearchPathForDirectoriesInDomains(.CachesDirectory, .UserDomainMask, true).first
             else
             {
                 fatalError("no cache directories found")
             }
             
-            return NSURL(fileURLWithPath: directory)
+            // We need to create a directory in `../Library/Caches folder`. Otherwise, trying to remove the Apple /Caches folder will always fails. Note that it's noticeable while testing on a device.
+            let cacheDirectory = directory + Constant.cacheDirectory
```

Comment:
This will just append the directory to make the path "Cachescom.vimeo.Caches" when I think we want "Caches/com.vimeo.Caches"

Can we make use of NSURL's api to append a path component so we get that `/` ?